js/coinsbit.js

```diff
@@ -0,0 +1,1103 @@
+        //     result: {
+        //       orderId: '8631519189',
+        //       market: 'ETH_USDT',
+        //       price: '3100',
+        //       side: 'buy',
+        //       type: 'limit',
+        //       timestamp: '1640254497.745',
+        //       dealMoney: '0',
+        //       dealStock: '0',
+        //       amount: '0.002',
+        //       takerFee: '0.002',
+        //       makerFee: '0.002',
+        //       left: '0.002',
+        //       dealFee: '0'
+        //     },
+        //     code: '200'
+        //  }
+        const data = this.safeValue (response, 'result');
+        return this.parseOrder (data);
+    }
+
+    async fetchOpenOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' fetchOrders requires a symbol argument');
+        }
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'market': market['id'],
+        };
+        const offset = this.safeValue (params, 'offset');
+        if (offset !== undefined) {
+            request['offset'] = offset;
+        }
+        if (limit !== undefined) {
+            request['limit'] = limit;
+        }
+        const response = await this.privatePostOrders (this.extend (request, params));
+        // {
+        //     success: true,
+        //     message: '',
+        //     result: {
+        //         limit: '50',
+        //         offset: '0',
+        //         total: '1',
+        //         result: [
+        //            {
+        //               id: '8633074431',
+        //               left: '0.002',
+        //               market: 'ETH_USDT',
+        //               amount: '0.002',
+        //               type: 'limit',
+        //               price: '2000.1892',
+        //               timestamp: '1640284932.136',
+        //               side: 'buy',
+        //               dealFee: '0',
+        //               takerFee: '0.002',
+        //               makerFee: '0.002',
+        //               dealStock: '0',
+        //               dealMoney: '0'
+        //           },
+        //           ...
+        //         ]
+        //     },
+        //     code: '200'
+        // }
+        const data = this.safeValue (response, 'result', []);
+        const orders = this.safeValue (data, 'result', []);
+        return this.parseOrders (orders, market, since, limit);
+    }
+
+    async fetchClosedOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        const request = {};
+        if (limit !== undefined) {
+            request['limit'] = limit;
+        }
+        const method = this.safeValue (params, 'method', 'account/order_history');  // account/order_history || account/order_history_list
+        if (method === 'account/order_history') {
```

Comment:
Let's do the same edit here as described above.

Reply:
Sorry, I doubt I can't guess (as above comment was about exception message). please tell once more, a bit more explicitly

Reply:
I mean replace `account/order_history` with `privatePostAccountOrderHistory`.